.github/workflows/codeql.yml

```
@@ -38,26 +79,42 @@ jobs:
+    steps:
+    - uses: actions/checkout@v6
+
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v4
+      with:
+        languages: csharp
+        build-mode: none
+
+    - name: Perform CodeQL Analysis
+      uses: github/codeql-action/analyze@v4
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)